internal/admin/dashboard/templates/index.html

```
@@ -253,6 +260,162 @@ <h2>User Settings</h2>
+                        </template>
+                    </select>
+                </label>
+
+                <label class="alias-form-field">
+                    <span>Description</span>
+                    <input type="text" class="filter-input" placeholder="What this guardrail is meant to do" x-model="guardrailForm.description">
+                </label>
+
+                <label class="alias-form-field">
+                    <span>User Path</span>
+                    <input type="text" class="filter-input" placeholder="/team/alpha" x-model="guardrailForm.user_path" aria-label="Guardrail user path">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)